PhysicsPackages/fixedSourcePhysicsPackage_class.f90

```diff
@@ -181,6 +181,7 @@ subroutine cycles(self, tally, N_cycles)
 
         ! Obtain particle from dungeon
         call self % thisCycle % copy(p, n)
+        p % showerID = n
```

Comment:
Same comment as before about the `copy`.

Reply:
Applied